Mathlib/Topology/MetricSpace/Katetov.lean

```diff
@@ -0,0 +1,123 @@
+/-
+Copyright (c) 2024 Luigi Massacci. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Luigi Massacci
+-/
+
+import Mathlib.Topology.MetricSpace.Basic
+import Mathlib.Topology.UniformSpace.Cauchy
+
+/-!
+# Katetov Maps
+
+In this file we define Katetov maps (i.e. one point extensions of a metric) and establish
```

Comment:
How does "one point extension of a metric" relate to the definition below? Please add some explanation for users like me who never used Katetov maps.